kollect-core/src/main/kotlin/kollect/arrow/instances/instances.kt

```diff
@@ -0,0 +1,14 @@
+package kollect.arrow.instances
+
+import arrow.instance
+import arrow.typeclasses.Monoid
+import kollect.InMemoryCache
+
+@instance(InMemoryCache::class)
+interface InMemoryCacheMonoidInstance: Monoid<InMemoryCache> {
```

Comment:
We need to add a test that depends on `arrow-test` and tests the monoid laws against these instances. You can see examples of such tests that extend `UnitSpec` on https://github.com/arrow-kt/arrow/blob/072ee06e72c29c962e5a20b263bbb8da8fdf6fe2/modules/core/arrow-data/src/test/kotlin/arrow/data/SortedMapKWTest.kt#L36